stackit/internal/services/argus/instance/resource_test.go

```diff
@@ -576,6 +599,47 @@ func TestMapAlertConfigField(t *testing.T) {
 			},
 			isValid: true,
 		},
+		{
+			description: "no global options",
```

Comment:
Seems like this test case should have an empty global options object instead of a nil one, otherwise it's just the same as the above test cases. Or am I missing something?